src/components/Aside/AddRestaurantModal.jsx

```diff
@@ -1,42 +1,38 @@
-                        <label htmlFor="category" className="text-caption">카테고리</label>
-                        <select name="category" id="category" required>
-                            <option value="">선택해 주세요</option>
-                            {categories.map((category) => (
-                                <option key={category.id} value={category.name}>{category.name}</option>
-                            ))}
-                        </select>
-                    </div>
-
-                    <div className={`${styles["form-item"]} ${styles["form-item--required"]}`}>
-                        <label htmlFor="name" className="text-caption">이름</label>
-                        <input type="text" name="name" id="name" required></input>
-                    </div>
-
-                    <div className={styles["form-item"]}>
-                        <label htmlFor="description" className="text-caption">설명</label>
-                        <textarea name="description" id="description" cols="30" rows="5"></textarea>
-                        <span className={`${styles["help-text"]} text-caption`}>메뉴 등 추가 정보를 입력해 주세요.</span>
-                    </div>
-
-                    <div className={styles["button-container"]}>
-                        <button className={`${styles.button} ${styles["button--primary"]} text-caption`}>추가하기</button>
-                    </div>
-                </form>
-            </div>
-        </div>
+        <Modal title="새로운 음식점" onClose={() => onChangeAddModal(null)}>
+            <form method="post" onSubmit={(event) => onChangeAddModal(event.target)}>
+                <div className={`${styles["form-item"]} ${styles["form-item--required"]}`}>
+                    <label htmlFor="category" className="text-caption">카테고리</label>
+                    <select name="category" id="category" required>
+                        <option value="">선택해 주세요</option>
+                        {categories.map((category) => (
+                            <option key={category.id} value={category.name}>{category.name}</option>
+                        ))}
+                    </select>
+                </div>
+                <div className={`${styles["form-item"]} ${styles["form-item--required"]}`}>
+                    <label htmlFor="name" className="text-caption">이름</label>
+                    <input type="text" name="name" id="name" required></input>
+                </div>
+                <div className={styles["form-item"]}>
+                    <label htmlFor="description" className="text-caption">설명</label>
+                    <textarea name="description" id="description" cols="30" rows="5"></textarea>
+                    <span className={`${styles["help-text"]} text-caption`}>메뉴 등 추가 정보를 입력해 주세요.</span>
+                </div>
+                <div className={styles["button-container"]}>
+                    <button className={`${styles.button} ${styles["button--primary"]} text-caption`}>추가하기</button>
```

Comment:
```suggestion
                    <button className={`${styles.button} ${styles["button--primary"]} text-caption`} onClick ={onChangeModal}>추가하기</button>
```
위에 함수를 onFormSubmit으로 바꿔서 이부분을 추가했습니다..! 이런식으로 해도 좋을 거 같아요

Reply:
분리해보니 정보를 추가하지 않고 추가버튼 눌렀을 때 그냥 닫히는 문제가 생기더군요... 조금 고민해봐야 될거 같아요